README.md

```diff
@@ -15,12 +15,11 @@ TODO
 
 ## Usage
 
-Using Gidari is a two step process:
+Document how to use the package with `gidari.Transport` 
 
-1. Create a configuraiton file to instruct the binary on how to make the RESful HTTP requests and where to store the data
-2. Run `gidari --config your_configuration.yml --verbose`
+### Command Mode
 
-### Configuration
+All the existing CLI logic stays here, remove the "Configuration" header.
```

Comment:
Again, we don't want to add this line exactly, this was just a placeholder. We need to document how to use Gidari in command mode.